bruno/github api/get repo contents.bru

```diff
@@ -0,0 +1,16 @@
+meta {
+  name: get repo contents
+  type: http
+  seq: 1
+}
+
+get {
+  url: https://api.github.com/repos/ls1intum/Artemis/contents/.browserslistrccc
```

Comment:
```suggestion
  url: https://api.github.com/repos/ls1intum/Artemis/contents/.browserslistrc
```
Add working example

Reply:
Also the result returns the file content but says the content is json (which is not true)
![image](https://github.com/user-attachments/assets/14801425-b726-418f-bfe3-27e85a82c0c7)